README.md

```diff
@@ -138,6 +138,9 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What is the extent of your experience with Promises and/or their polyfills?
 * What are the pros and cons of using Promises instead of callbacks?
 * What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
+* Explain mutability and immutability on example of JS basic types (string, object, array, number, etc.)
```

Comment:
Maybe fix this up to be more like:
> Explain mutable and immutable objects.
  * What is an example of an immutable object in JavaScript?
  * What are the pros and cons of immutability?
  * How can you achieve immutability in your own code?

Referencing the basic types seems to give away an answer you could ask for immutable objects.